packages/paste-core/components/menu/src/MenuGroup.tsx

```diff
@@ -1,35 +1,44 @@
 import * as React from 'react';
 import {safelySpreadBoxProps, Box} from '@twilio-paste/box';
 import {Text} from '@twilio-paste/text';
-import {MenuItemVariants} from './constants';
+import {MenuItemVariants, DEFAULT_ELEMENT_NAME} from './constants';
 import type {MenuItemVariant, MenuGroupProps} from './types';
 
 export const MenuGroupContext = React.createContext<MenuItemVariant>(MenuItemVariants.DEFAULT);
 
-const MenuGroup = React.forwardRef<HTMLDivElement, MenuGroupProps>(({label, icon, children, ...props}, ref) => {
-  return (
-    <MenuGroupContext.Provider value={MenuItemVariants.GROUP_ITEM}>
-      <Box {...safelySpreadBoxProps(props)} role="presentation" aria-label={label} textDecoration="none" ref={ref}>
-        <Box display="flex" alignItems="center" paddingX="space70" paddingY="space30" cursor="default">
-          {React.isValidElement(icon) ? (
-            <Box flexShrink={0} size="sizeIcon30">
-              {React.cloneElement(icon, {color: 'colorTextIcon'})}
-            </Box>
-          ) : null}
-          <Text
-            as="span"
-            color="colorText"
-            role="presentation"
-            fontWeight="fontWeightBold"
-            paddingLeft={icon != null ? 'space20' : undefined}
-          >
-            {label}
-          </Text>
+const MenuGroup = React.forwardRef<HTMLDivElement, MenuGroupProps>(
+  ({label, icon, children, element = DEFAULT_ELEMENT_NAME, ...props}, ref) => {
```

Comment:
It's called menu group below 👇

Reply:
Ah yeah, I mean like `element="MENU_GROUP"` right here exactly, and then `element={element}` below since we don't need to join strings for any other reason

Reply:
Ah yeah, I mean like element="MENU_GROUP" right here exactly, and then element={element} below since we don't need to join strings for any other reason